tests/integration/modules/ch_backup.py

```diff
@@ -198,6 +198,9 @@ def __init__(self, context: ContextT, node_name: str, timeout: int = 300) -> Non
         protocol = context.ch_backup["protocol"]
         self._cmd_base = f"timeout {timeout} {CH_BACKUP_CLI_PATH} --protocol {protocol} --insecure  --config {self._config_path}"
 
+    def get_version(self) -> str:
```

Comment:
What a difference with `version` method below? Looks like the same.

Reply:
Yes, you right. Missed it. Removed it in the last commit.